libs/execution_context/ebpf_native.c

```diff
@@ -410,9 +406,10 @@ void
 ebpf_native_release_reference(_In_opt_ _Post_invalid_ ebpf_native_program_t* binding_context)
```

Comment:
[Not due to current changes] Here too the name of the function is a little misleading as release_reference implies something to do with reference count. Consider renaming.

Reply:
`ebpf_native_release_reference` does release reference on the native module, though it also does some clean up (which is opaque for the code in `ebpf_program.c`.